src/main/resources/templates/review.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="https://www.thymeleaf.org/" lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
```

Comment:
I think this was overlooked, shouldn't this be changed?

Reply:
Fixed in https://github.com/2020-Spring-Cohort/reviews-site-models/pull/37/commits/f4db108cdc1536f46444657d7d087d47186ad20a